bigbluebutton-html5/imports/api/guest-users/server/methods/updatePositionInWaitingQueue.js

```diff
@@ -0,0 +1,35 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import RedisPubSub from '/imports/startup/server/redis';
+import Logger from '/imports/startup/server/logger';
+import GuestUsers from '/imports/api/guest-users/';
+
+
+const REDIS_CONFIG = Meteor.settings.private.redis;
+const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
+const EVENT_NAME = 'UpdatePositionInWaitingQueueReqMsg';
+
+export default function updatePositionInWaitingQueue(meetingId) {
+
+    check(meetingId, String); 
+
+    const guestUsers = GuestUsers.find({
+        meetingId: meetingId,
+        approved: false,
+        denied: false,
+    }).fetch();
+    
+    check(guestUsers, Array);
+    try {
+
+        const guests = guestUsers.map(guest => ({intId: guest.intId, idx: String(guestUsers.indexOf(guest)+1)}));
```

Comment:
Although this is OK, I recommend to push this kind of processing to akka-apps to avoid using the server's Node process.